diff --git a/dist/OfflineIcon.svelte.d.ts b/dist/OfflineIcon.svelte.d.ts
index 7014575ab8d20e54323f75598d9f84faf262c998..e350be26ac2cb11a5103075448c740fdb1790ad4 100644
--- a/dist/OfflineIcon.svelte.d.ts
+++ b/dist/OfflineIcon.svelte.d.ts
@@ -82,7 +82,7 @@ export declare type IconifyRenderMode = 'style' | 'bg' | 'mask' | 'svg';
 /**
  * Mix of icon properties and HTMLElement properties
  */
-export declare type IconProps = IconifyElementProps & IconifyIconProps;
+export declare type IconProps = IconifyElementProps & IconifyIconProps & {class: string;};
 
 /**
  * Svelte component
diff --git a/dist/index.d.ts b/dist/index.d.ts
index 469dc04826c523bbd877503aa16e98815762453d..d32325ca13dc494a0849fd7784905b9715fd4a55 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -315,7 +315,7 @@ export declare interface IconifyStorageFunctions {
 /**
  * Mix of icon properties and HTMLElement properties
  */
-export declare type IconProps = IconifyElementProps & IconifyIconProps;
+export declare type IconProps = IconifyElementProps & IconifyIconProps & {class: string;};
 
 /**
  * List available icons