diff --git a/index.d.ts b/index.d.ts
index 610123da3dc418267b4b5fe0636bb0d9314e1312..7de2b3e3e39e9090292ab2e1074f4ced6c00e498 100755
--- a/index.d.ts
+++ b/index.d.ts
@@ -1396,7 +1396,7 @@ export class ImageOverlay extends Layer {
     setUrl(url: string): this;
 
     /** Update the bounds that this ImageOverlay covers */
-    setBounds(bounds: LatLngBounds): this;
+    setBounds(bounds: LatLngBoundsExpression): this;
 
     /** Changes the zIndex of the image overlay */
     setZIndex(value: number): this;
@@ -1419,7 +1419,7 @@ export class ImageOverlay extends Layer {
     options: ImageOverlayOptions;
 }
 
-export function imageOverlay(imageUrl: string, bounds: LatLngBoundsExpression, options?: ImageOverlayOptions): ImageOverlay;
+export function imageOverlay(imageUrl: string | HTMLImageElement, bounds: LatLngBoundsExpression, options?: ImageOverlayOptions): ImageOverlay;
 
 export type SVGOverlayStyleOptions = ImageOverlayStyleOptions;
 